changelogs/fragments/filetree_create_bugfix.yml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - When exporting job templates it was failing when missing some input information.
+  - When importing the exported settings, fields like `AUTOMATION_ANALYTICS_LAST_GATHER: 2023-07-17T13:22:06.445818Z` caused problems with the multiple `:`.
```

```suggestion
  - "When importing the exported settings, fields like `AUTOMATION_ANALYTICS_LAST_GATHER: 2023-07-17T13:22:06.445818Z` caused problems with the multiple `:`."
```

The pre-commit is failing because of this line because theres a `:` it interprets it weirdly. Putting in quotes like this *should* fix it